src/main/resources/templates/forum-mainpage.html

```diff
@@ -0,0 +1,107 @@
+            integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+            crossorigin="anonymous"></script>
+    <meta charset="utf-8">
+    <title>Foren</title>
+    <th:block th:fragment="headcontent">
+        <style>
+            main > div:first-child {
+                position: relative;
+                margin-bottom: 1rem;
+            }
+
+            main > div:first-child img {
+                border-radius: 1rem;
+                border: 1px solid black;
+            }
+
+            main > div:first-child > h2 {
+                position: absolute;
+                bottom: 0;
+                margin: 1rem;
+                color: white;
+                text-shadow: 0 0 3px black;
+            }
+
+            .card {
+                box-shadow: 0 14px 28px rgba(0, 0, 0, 0.15), 0 10px 10px rgba(0, 0, 0, 0.12);
+                text-align: center;
+                width: 200px;
+                padding-left: 20px;
+                padding-right: 20px;
+                margin: 10px;
+                float: left;
+            }
+
+            @media (min-width: 992px) {
+                main > div:first-child > h2 {
+                    font-size: 2.4rem;
+                }
+            }
+
+            @media (max-width: 576px) {
+                main > div:first-child > h2 {
+                    font-size: 1.0rem;
+                }
+            }
+        </style>
+    </th:block>
+</head>
+<body>
+<header>
+    <nav class="navigation navigation-secondary" is="mops-navigation" th:fragment="navigation">
+        <ul>
+            <li class="">
+                <a href="#welcome" th:href="@{./}">Dashboard</a>
+            </li>
+            <li class="active">
+                <a href="#welcome" th:href="@{./my-forums}">Meine Foren</a>
+            </li>
+            <li class="">
+                <a href="#welcome" th:href="@{./profile}">Profil</a>
+            </li>
+        </ul>
+    </nav>
+</header>
+<main th:fragment="bodycontent">
+    <div style="padding: 30px;">
+        <h1 th:text="${forum.getTitle()}">Algorithmen und Datenstrukturen</h1>
+        <br>
+        <div th:text="${forum.getDescription()}">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Magni
+            laboriosam dolor ratione dolorum delectus. Tempore magni consequatur unde quam dolore amet! Beatae
+            accusantium in aspernatur natus labore facilis animi harum!
+        </div>
+        <br>
+        <div th:each="topic : ${topics}" class="justify-content-center">
+            <div class="card">
+                <div class="card-body">
+                    <h5 th:text="${topic.getTitle()}" class="card-title">Title</h5>
+                    <button class="btn btn-primary"
+                            th:onclick="location.href = '/my-forums/' + [[${forum.getId().getId().toString()}]] + '/' + [[${topic.getId().getId().toString()}]]"
```

Comment:
ist das toString wirklich nötig?

Reply:
Thymeleaf crashes when trying to use Long,